File: lab4 mpp/TeledonDomain/CazCaritabil.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace teledon.domain
{
    public class CazCaritabil : Entity<long>
    {

        public String Nume { get; set; }
        public int SumaAdunata { get; set; }
       
        public CazCaritabil(String nume)
        {
            this.Nume = nume;
            SumaAdunata = 0;
        }
        public CazCaritabil(String nume, int suma)
        {
            this.Nume = nume;
            this.SumaAdunata = suma;
        }
        public void AdaugaDonatie(int Suma)
        {
            this.SumaAdunata += Suma;
        }

        public override string ToString()
        {
            return $"{Nume} , {SumaAdunata}";
        }

    }
}

File: GUImpp/TeledonModel/src/main/java/teledon/model/CazCaritabil.java
package teledon.model;

public class CazCaritabil extends Entity<Long> {


    public String nume;
    public int SumaAdunata;

    public String getNume() {
        return nume;
    }

    public void setNume(String nume) {
        this.nume = nume;
    }

    public int getSumaAdunata() {
        return SumaAdunata;
    }

    public void setSumaAdunata(int sumaAdunata) {
        SumaAdunata = sumaAdunata;
    }

    public CazCaritabil(String nume, int sumaAdunata) {
        this.nume = nume;
        SumaAdunata = sumaAdunata;
    }
    public CazCaritabil(String nume) {
        this.nume = nume;
        SumaAdunata = 0;
    }

    public CazCaritabil() {

    }

    @Override
    public String toString() {
        return this.nume + " " + this.SumaAdunata;
    }

}
